frontend/src/entry/CreateEntry.tsx

```diff
@@ -130,6 +136,34 @@ export default function CreateEntry({ createEntryKind, onBackClick }: CreateEntr
     )
   }
 
+  const templateLabel = () => {
+    return (
+      <Stack direction='row' justifyContent='center' alignItems='center' spacing={1}>
+        <FolderCopy />
+        <Stack sx={{ my: 1 }}>
+          <Typography fontWeight='bold'>Templating</Typography>
+          <Typography variant='caption'>
+            {`Allow this to be used as a template for another ${EntryKindLabel[createEntryKind]}`}
+          </Typography>
+        </Stack>
+      </Stack>
+    )
+  }
+
+  const ungovAccessLabel = () => {
```

Comment:
`const ungovAccessLabel = useCallback(() => {`

I'd also consider renaming this to `ungovernedAccessLabel` as well